Turning on USB logging interferes with TX/RX pins on Xiao BLE

I have this issue where turning on USB logging via `CONFIG_ZMK_USB_LOGGING=y` prevents the matrix pin I have at RX (D7/P1.12) from working. I am guessing this is because they are assigned to UART pins at https://github.com/zmkfirmware/zephyr/blob/v3.2.0%2Bzmk-fixes/boards/arm/seeeduino_xiao_ble/seeeduino_xiao_ble-pinctrl.dtsi#L7 and logging turns it on via https://github.com/zmkfirmware/zephyr/blob/v3.2.0%2Bzmk-fixes/boards/arm/seeeduino_xiao_ble/seeeduino_xiao_ble.dts#L17.

I see some shield configs deleting these properties, e.g. https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/hummingbird/hummingbird.overlay#L13. Should this be something that every shield does, or would it be better to solve it at the `seeeduino_xiao_ble.overlay` level?

Actually I didn't have any luck with enabling USB logging with those nodes deleted, the stock `hummingbird` shield fails building too:
```sh
> west build -p -d build/hummingbird -b seeeduino_xiao_ble -- -DSHIELD=hummingbird -DCONFIG_ZMK_USB_LOGGING=y
[...]
/home/.../zmk/zephyr/include/zephyr/device.h:104:39: error: '__device_dts_ord_DT_CHOSEN_zephyr_console_ORD' undeclared here (not in a function)
[...]
```

--- a/app/boards/seeeduino_xiao_ble.overlay
+++ b/app/boards/seeeduino_xiao_ble.overlay
@@ -19,6 +19,8 @@
     };
 };
 
+&xiao_serial { status = "disabled"; };
+
 &adc {
     status = "okay";
 };

